benchmark/benchmarkTests.go

```diff
@@ -146,8 +153,11 @@ func SociFullRun(
 	}
 	defer cleanupTaskSecondRun()
 	log.G(ctx).WithField("benchmark", "RunTaskTwice").WithField("event", "Start").Infof("Start Run Task Twice")
+	runTask2Start := time.Now()
 	cleanupRunSecond, err := sociContainerdProc.RunContainerTaskForReadyLine(ctx, taskDetailsSecondRun, readyLine)
+	runTask2Duration := time.Since(runTask2Start)
 	log.G(ctx).WithField("benchmark", "RunTaskTwice").WithField("event", "Stop").Infof("Stop Run Task Twice")
+	b.ReportMetric(float64(runTask2Duration.Milliseconds()), "runTask2Duration")
```

Comment:
nit: `runTaskWithoutOverhead`

Reply:
Used LocalTask instead of ''runTaskWithoutOverhead"